client/src/components/lib/pageStyles.ts

```diff
@@ -19,7 +19,7 @@ along with Home Lights.  If not, see <http://www.gnu.org/licenses/>.
 
```

Comment:
Have you thought about using styledComponents? https://styled-components.com/

Reply:
I think it's compatible with the material-ui lib

Reply:
I thought about it, but decided that Material UI's CSS library is good enough. I don't think there's going to be much more styling in the app than what's already here.